server/controllers/finance/generalLedger/index.js

```diff
@@ -69,20 +78,41 @@ function listAccounts(req, res, next) {
  * @function getlistAccounts
  * get list of accounts
  */
-function getlistAccounts() {
+function getlistAccounts(periodsId) {
+  let jointure = '';
+  let headSql = '';
+
+  if(periodsId){
+    periodsId.forEach(function (period) {
+      headSql += `
+        , periode${period.id}.balance AS balance${period.number}
+      `;
+
+      jointure += `
```

Comment:
I don't understand this code ... why do we need to LEFT JOIN `general_ledger` on `period_total`?  Why can't we just pull all the data from `period_total`?